### DISCUSSION TITLE
Errore nel sistema

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#646] [Richieste di aiuto] [answered]
Salve,
quando firmo e metto i voti non appare il simbolo aggiungi/modifica e orale, scritto e pratico
nonostante è finito il quadrimestre e oggi abbiamo chiuso lo scrutinio.
Controllando i parametri risulta tutto giusto come devo fare?

**Comment by Davide031** [comment]
Chiedo aiuto a @trinko  e ai suoi collaboratori @zerai @MatteoGheza

**Comment by trinko** [comment]
Direi che è dovuto a un problema nella configurazione.
Se non li hai già controllati, guarda i parametri con le date di inizio e fine anno scolastico, e quelli con le date di fine dei periodi.

Altra cosa da controllare sono le impostazioni degli scrutini, dalla pagina **SCUOLA > SCRUTINI**: vedi se i periodi successivi sono definiti e, in tal caso, se le date sono corrette.

**Reply by Davide031** [reply]
Sono tutti giusti

**Comment by trinko** [comment]
Può essere dovuto alla presenza, per la stessa classe, di uno scrutinio del periodo successivo, magari eseguito per un test.

Puoi verificarlo usando, dal pannello di controllo, PHPMyAdmin. Inserisci nella finestra SQL il seguente comando:
```
SELECT * 
FROM gs_scrutinio s, gs_classe c
WHERE s.classe_id=c.id AND s.periodo<>'P'
AND c.anno=1 AND c.sezione='A';
```
**Modifica l'ultima riga inserendo l'anno e la sezione della classe in cui hai il problema** (nel codice si fa riferimento alla 1A).

Se vengono visualizzati dei dati, il problema è quello.

**Reply by Davide031** [reply, marked as answer]
localhost/my_re2024/		https://s577.altervista.org/phpmyadmin/index.php?route=/database/sql&db=my_re2024

   Mostro le righe 0 -  1 (2 del totale, La query ha impiegato 0.0005 secondi.)


SELECT * 
FROM gs_scrutinio s, gs_classe c
WHERE s.classe_id=c.id AND s.periodo<>'P'
AND c.anno=1 AND c.sezione='A';


id	classe_id	creato	modificato	periodo	data	inizio	fine	stato	dati	visibile	sincronizzazione	id	sede_id	corso_id	coordinatore_id	segretario_id	creato	modificato	anno	sezione	ore_settimanali	gruppo	
2	1	2023-11-28 18:18:26	2024-02-09 16:16:13	F	2023-11-28	18:00:00	18:30:00	1	a:15:{s:6:"alunni";a:10:{i:0;i:37;i:1;i:13;i:2;i:2...	2024-06-13 12:00:00		1	1	1	2	4	2023-09-28 17:12:39	2023-12-11 15:49:20	1	A	30	NULL	
3	1	2024-01-21 15:48:16	2024-01-21 15:48:16	S	NULL	NULL	NULL	N	a:0:{}	2024-02-13 12:00:00		1	1	1	2	4	2023-09-28 17:12:39	2023-12-11 15:49:20	1	A	30	NULL	
Mi esce questo

**Reply by Davide031** [reply]
È quello me lo ha riferito il coordinatore ho provato a risolvere dal pannello admin ma non riesco come posso fare?

**Reply by Davide031** [reply]
ho cercato di sistemare provando a cambiare l'inizio dello scrutinio finale ma nulla

**Comment by Davide031** [comment]
Ho sistemato il collega lo ha lasciato aperto.
Grazie mille

**Comment by trinko** [comment]
C'è lo scrutinio finale che risulta aperto.

Dovrebbe risolversi in questo modo:
- come amministratore, vai alla pagina **SCUOLA > SCRUTINI**, seleziona il periodo dello scrutinio finale e imposta la data di inizio scrutinio al giorno di oggi;
- come staff, vai alla pagina **COORDINATORE > SCRUTINIO** per la classe che da' problemi; dovresti vedere lo scrutinio aperto al passo 1 (presenze dei docenti);
- clicca sul pulsante in basso a sinistra **ANNULLA APERTURA SCRUTINIO**;
- come amministratore, torna alla pagina **SCUOLA > SCRUTINI**, seleziona il periodo dello scrutinio finale e imposta la data di inizio scrutinio alla data prevista per giugno.

**Comment by trinko** [comment]
Bene, lascio il messaggio che avevo appena scritto nel caso fosse utile a qualcun altro.
